components/OddsCalculatorDialog/SkillOddsRow.tsx

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import { clone, find } from 'lodash';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import Collapsible from 'react-native-collapsible';
+import { t } from 'ttag';
+
+import typography from '../../styles/typography';
+import { ChaosBag, ChaosTokenType, SKILL_COLORS, SKILL_COLORS_LIGHT, SkillCodeType } from '../../constants';
+import { flattenChaosBag } from '../campaign/campaignUtil';
+import ArkhamIcon from '../../assets/ArkhamIcon';
+import PlusMinusButtons from '../core/PlusMinusButtons';
+import { COLORS } from '../../styles/colors';
+import { add, binomdist, formatPercentageText, subtract } from './oddsHelper';
+import { s } from '../../styles/space';
+
+export interface SkillOddsRowProps {
+  chaosBag: ChaosBag;
+  stat: number;
+  specialTokenValues: { token: ChaosTokenType; value: number; raw_value: string | null }[];
+  type: SkillCodeType;
+  testDifficulty: number;
+}
+
+type Props = SkillOddsRowProps;
+
+interface State {
+  boosts: {[skill in SkillCodeType]: number};
```

Comment:
spaces after/before {}

Reply:
Very strange. My IDE shows this line to have the correct spaces but I have no uncommited changes and this still shows it without the spaces.